core/app/models/spree/export.rb

```
@@ -194,5 +202,26 @@ def set_default_format
+      %w[created_at completed_at].each do |field|
+        DATE_FIELD_SUFFIX_TO_PARSER.each do |suffix, parser|
+          key = "#{field}#{suffix}"
+          params[key] = send(parser, params[key]) if params[key].present?
+        end
+      end
+
+      params
+    end
+
+    def parse_to_beginning_of_day(value)
+      value.to_date&.in_time_zone(store.preferred_timezone).beginning_of_day.iso8601 rescue ""
```

RuboCop (1.76.1)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)